x-pack/legacy/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -85,11 +85,25 @@ const traceListColumns: Array<ITableColumn<ITransactionGroup>> = [
   },
   {
     field: 'impact',
-    name: i18n.translate('xpack.apm.tracesTable.impactColumnLabel', {
-      defaultMessage: 'Impact'
-    }),
+    name: (
+      <EuiToolTip
+        content={i18n.translate(
+          'xpack.apm.tracesTable.impactColumnDescription',
+          {
+            defaultMessage:
+              "The most used and slowest endpoints in your service. It's calculated by taking the relative average duration times the number of transactions per minute."
+          }
+        )}
+      >
+        <>
+          {i18n.translate('xpack.apm.tracesTable.impactColumnLabel', {
+            defaultMessage: 'Impact'
+          })}
+        </>
+      </EuiToolTip>
+    ),
```

Comment:
EUI has an [example in their tables](https://elastic.github.io/eui/#/display/tables) docs where they add a small icon to indicate that there's some more information on the table name.

<img width="969" alt="Screenshot 2019-10-08 at 20 20 51" src="https://user-images.githubusercontent.com/4104278/66422146-73ce0e80-ea09-11e9-9172-8f1fcefb1328.png">


```
name: (
          <EuiToolTip content="Their mascot is the Octokitty">
            <span>
              Github{' '}
              <EuiIcon
                size="s"
                color="subdued"
                type="questionInCircle"
                className="eui-alignTop"
              />
            </span>
          </EuiToolTip>
        )
```

Would we be able to add this in as a small visual enhancement too?

Reply:
Added in 684e12dd31e.